network/syncing/syncer.go

```diff
@@ -207,9 +206,9 @@ func (s *syncer) getDecidedByRange(
 			})
 			logger.Debug("received and processed history batch",
 				zap.Int64("tail", int64(tail)),
-				zap.Duration("duration", time.Since(start)),
+				logging.DurationNano(start),
 				zap.Int("results_count", len(msgs)),
-				zap.String("results", s.formatSyncResults(msgs)),
+				logging.Results(msgs),
```

Comment:
too vague. need to be based on what the result type is

Reply:
would something like "sync_results" be good?

Reply:
yeah, better